model/registry/deprecated/system.yaml

```diff
@@ -0,0 +1,20 @@
+groups:
+  - id: attributes.system.deprecated
```

Comment:
I guess we want to be consistent here? Prefix or suffix with `deprecated`? We probably should also add guidance on how to deprecate things like this in the future.

Reply:
good catch! I'll update contributing guide as a part of #740